content/docs/aws/storage.md

```diff
@@ -27,11 +26,89 @@ You can go to the [Amazon EFS console](https://us-west-2.console.aws.amazon.com/
 Please pay special attention to the Security Groups, and make sure that traffic to NFS port 2049 is allowed.
 Then you will get a file system ID and you can use it to create persistent volumes and persistent volume claims.
 
-```shell
-cd ${KF_DIR}/ks_app
-export COMPONENT=efs-storage
-ks generate aws-efs-pv ${COMPONENT} --efsId=${your_file_system_id}
-ks apply default -c ${COMPONENT}
+For EKS user using eksctl, you can choose `ClusterSharedNodeSecurityGroup`.
+<img src="/docs/images/aws/efs-create.png"
+  alt="Amazon EFS Create"
+  class="mt-3 mb-3 border border-info rounded">
+
+
+You will need to create storage class for the first time.
```

Comment:
Can we link to a doc that tells the reader what to do with the YAML files? For example, do they paste the content into the EFS Console, or do they put the YAML files in a particular directory in their Kubeflow deployment? (Please ignore this comment if I've overlooked something and the answer is obvious to the reader.)

Reply:
This makes sense. I think I assumed user apply this yaml. Thanks for giving the view from anonymous. I add more guidance like `save this file as xx.yaml, run kubectl apply -f xx.yaml`